webrtc.html

```diff
@@ -9849,6 +9853,40 @@ <h2>Methods</h2>
         </section>
       </div>
     </section>
+    <section>
+      <h3>canInsertDTMF update</h3>
+      <p>To <dfn>update the [[\CanInsertDTMF]]</dfn> slot of an <code><a>RTCDTMFSender</a></code>
+      instance <var>dtmfSender</var>, the user agent MUST queue a task that runs the following steps:</p>
+      <ol>
+        <li>Let <var>sender</var> be the <code><a>RTCRtpSender</a></code> associated with
+        <var>dtmfSender</var>.</li>
+        <li>Let <var>transceiver</var> be the <code><a>RTCRtpTransceiver</a></code>
+        associated with <var>sender</var>.</li>
+        <li>Let <var>connection</var> be the <code><a>RTCPeerConnection</a></code> associated with
+        <var>transceiver</var>.</li>
+        <li>Check whether <var>connection</var>'s <a>[[\ConnectionState]]</a> slot
+        has the value <code>"connected"</code>. If not, set <var>dtmfSender</var>'s
+        <a>[[\CanInsertDTMF]]</a> slot to <code>false</code> and abort these steps.</li>
```

Comment:
Maybe simpler: *"If [[ConnectionState]] is not "connected", return true"* ?

Applies throughout.

Reply:
Yes, I like that approach (and thus removing the internal slot, and just have the algorithm return true or false).